### DISCUSSION TITLE
Does earthpy.appeears support downloading from an existing URL (previously submitted request)?

### DISCUSSION THREAD

**Discussion by edchanonline** [earthlab-education/Earth-Analytics-2023-01-Intro#422] [Q&A04 Raster Data]
We may have discussed this, but I don't remember. Does `earthpy.appeears` support downloading from an existing URL (previously submitted request)? I took a look at [this file](https://github.com/earthlab/earthpy/blob/ae32522290f89a46e8b87639865b22561ae403db/earthpy/appeears.py), but it doesn't look like this use case is one of the supported ones based on below's description. Do you have alternative suggestions? Thank you!

```

	Parameters
	----------
	download_key : str, optional
		Label used in data_dir and as the API job label
	ea_dir : pathlike, optional
		Replacement directory for ~/earth-analytics
	product : str
		A product code from 
		https://appeears.earthdatacloud.nasa.gov/products
	layer : str
		A layer code from 
		https://appeears.earthdatacloud.nasa.gov/products
	start_date : str
		Start date for data subset, as 'MM-DD-YYYY' 
		or 'MM-DD' if recurring
	end_date : str
		End date for data subset, as 'MM-DD-YYYY' 
		or 'MM-DD' if recurring
	recurring : bool
		Whether the date range recurs each year. 
		Requires year_range.
	year_range : str
		Year range for recurring dates, as '[YYYY,YYYY]'
	polygon: gpd.GeoDataFrame
		The spatial boundary to subset
```

**Comment by eculler** [comment]
@eggvoice no that is not supported yet (feel free to make a PR! it's in the misspelled apppears branch of earthpy). Right now, you could potentially hack it by modifying the .appeears_taskid file to contain the task id for the download. I've gotten that to work before but haven't tested it thoroughly.

**Reply by edchanonline** [reply]
Noted, thank you!!
